app/_locales/en/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```

Comment:
The full picture:

**Category**
Productivity

**Keywords**
- block, 
- comments,
- toxic,
- youtube,
- Reddit,
- offensive comments

**Title**
Behave! — Offensive Comments Blocker

**Short Title**
Behave!

**Tag Line**
Hides comments on YouTube, Reddit & Twitter depending on the desired level of toxicity and converts them into harmless emojis.

**Description**
Tired of offensive comments on your favorite social networks and media sites? Behave! allows you to set a level of toxicity you are willing to tolerate and replaces comments that exceed it with graphical emojis.

You are able to unhide any comment and read it. Don’t like it? Hide it again. You are in control now.

Click the Behave! icon in the Chrome toolbar and use a slider to pick the toxicity level according to your current mood.

Behave! works on:
YouTube
Reddit
Twitter

English is the only supported language at the moment.